CMakeLists.txt

```diff
@@ -365,6 +365,7 @@ if(WITH_SWIG)
     find_program(SWIG_EXECUTABLE
         NAMES swig
         PATHS
+	      ./swig/swigwin-3.0.12
```

Comment:
for the CI it would even be good to just specify the parameter manually -DSWIG_EXECUTABLE=<path to swig> during the configure stage. Adding now another long list of possible locations based on ./swig/.. is maybe not the best idea

Reply:
Indeed, good idea - thanks.

Reply:
I've managed to add SWIG to the path on the Windows CI in a clean way now and reverted the change to `CMakeLists.txt`.